SAS/linear-regression.qmd

```diff
@@ -1,32 +1,68 @@
 ---
 title: "Linear Regression"
 output: html_document
-date: "2023-04-22"
+date: Sys.date()
```

Comment:
Putting R code here shouldn't work. Try for example the following. 
For more on date handling, check out: https://quarto.org/docs/reference/dates.html
```suggestion
date: last-modified
date-format: D MMMM, YYYY
```